content/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/count.md

```diff
@@ -21,6 +21,16 @@ _**Output data type:** Integer_
 count(column: "_value")
 ```
 
+{{% note %}}
+#### Count empty tables
+`count()` will return `0` for empty tables, but to include empty tables in your data,
+you must set the following parameters for specific functions:
```

Comment:
`count()` returns `0` for an empty table. 
To include (column and row headers?) for an empty table, set the following parameters for (which?) specific functions: